brainscore/metrics/transformations.py

```diff
@@ -170,11 +170,13 @@ class Defaults:
         stratification_coord = 'object_name'  # cross-validation across images, balancing objects
         unique_split_values = False
         random_state = 1
+        preprocess_indices = None
 
     def __init__(self,
                  splits=Defaults.splits, train_size=None, test_size=None,
                  split_coord=Defaults.split_coord, stratification_coord=Defaults.stratification_coord, kfold=False,
-                 unique_split_values=Defaults.unique_split_values, random_state=Defaults.random_state):
+                 unique_split_values=Defaults.unique_split_values, random_state=Defaults.random_state,
+                 preprocess_indices=Defaults.preprocess_indices):
```

Comment:
```suggestion
                 preprocess_indices=None):
```

Reply:
Removing it from `Defaults` means that I need to call an instance of Splits() when accessing it in the `CrossValidation(Transformation)` like this:
```
class CrossValidation(Transformation):
    def __init__(self, *args, split_coord=Split.Defaults.split_coord,
                 stratification_coord=Split.Defaults.stratification_coord,
                 preprocess_indices=Split()._preprocess_indices, **kwargs):
        self._split_coord = split_coord
        self._stratification_coord = stratification_coord
        self._split = Split(*args, split_coord=split_coord, stratification_coord=stratification_coord, **kwargs)
        self._logger = logging.getLogger(fullname(self))
        self._preprocess_indices = preprocess_indices
```

I wonder whether it would be better to leave it in `Defaults` maybe?